src/events/guild/scheduled-events/guildScheduledEventCreate.ts

```diff
@@ -0,0 +1,36 @@
+import { GuildAuditLogs, GuildScheduledEvent, Permissions } from 'discord.js';
+import Event from "../../../structures/Event";
+import LoggingManager from "../../../utils/managers/LoggingManager";
+
+const loggingManager: LoggingManager = new LoggingManager();
+
+export default class extends Event {
+    constructor(...args: any[]) {
+        // @ts-ignore
+        super(...args, {});
+    }
+
+    async run(scheduledEvent: GuildScheduledEvent) {
+        if(!scheduledEvent.guild?.me?.permissions.has(Permissions.FLAGS.VIEW_AUDIT_LOG)) return;
+
+        let msg: string = "";
+        const logs: GuildAuditLogs<"GUILD_SCHEDULED_EVENT_CREATE"> = await scheduledEvent.guild?.fetchAuditLogs({ limit: 1, type: "GUILD_SCHEDULED_EVENT_CREATE" });
+        const first = logs.entries.first();
+        if(!first) return;
+        
+        const { executor, createdTimestamp } = first;
+        if(Date.now() < createdTimestamp + 3000) {
+            msg = await this.client.bulbutils.translate("event_guild_scheduled_event_create_moderator", scheduledEvent.guild?.id, {
+                scheduledEvent,
+                moderator: executor,
+            });
+        }
+
+        if(!msg)
+            msg = await this.client.bulbutils.translate("event_guild_scheduled_event_create", scheduledEvent.guild?.id, {
```

Comment:
can the message include more info on the event
- name
- description
- start time / end time (if prsent)

Reply:
done, it also only adds the description if there is one (like start/end time)